src/allotropy/parsers/cfxmaestro/constants.py

```diff
@@ -0,0 +1,8 @@
+from allotropy.allotrope.models.adm.pcr.benchling._2023._09.qpcr import ContainerType
+
+DISPLAY_NAME = "CFX Maestro"
```

Comment:
@Preston-O-Benchling, can we change the display name of the adapter to Bio-Rad CFX Maestro so we are capturing the company name that produces the software.